packages/auth/src/AuthProvider.tsx

```diff
@@ -21,7 +21,7 @@ export interface AuthContextInterface {
    */
   authToken: string | null // @WARN! deprecated, will always be null
   /* The current user's data from the `getCurrentUser` function on the api side */
-  currentUser: null | CurrentUser
+  currentUser?: null | CurrentUser
```

Comment:
I don't think this change is correct, since we don't allow currentUser to be undefined. It's almost either null or a value.